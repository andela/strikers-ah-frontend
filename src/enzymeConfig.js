import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import 'jest-localstorage-mock';

export default Enzyme.configure({
  adapter: new Adapter(),
});
