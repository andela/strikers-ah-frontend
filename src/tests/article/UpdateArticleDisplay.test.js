import React from 'react';
import { shallow } from 'enzyme';
import '../../enzymeConfig';

import {
  UpdateMessageDisplay,
  mapStateToprops,
} from '../../components/article/UpdateMessageDisplay';

const articleUpdate = {
  message: 'fafasfsaf',
  error: 'fafafa',
};
const props = {
  sweetalert: jest.fn(),
  articleUpdate,
  article: {
    error: 'error',
    article: [],
  },
};

const Wrapper = shallow(<UpdateMessageDisplay {...props} />);

describe('test for `UpdateMessageDisplay` component', () => {
  test('should render `UpdateMessageDisplay`', () => {
    expect(Wrapper).toMatchSnapshot();
  });

  test('should test `mapstatetoprops`', () => {
    const mockedState = {
      Article: {
        article: {
          articleUpdate: {},
        },
      },
    };

    const state = mapStateToprops(mockedState);
    expect(state).toBeTruthy();
  });
});
