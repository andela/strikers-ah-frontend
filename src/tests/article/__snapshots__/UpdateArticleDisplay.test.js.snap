// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for \`UpdateMessageDisplay\` component should render \`UpdateMessageDisplay\` 1`] = `ShallowWrapper {}`;
